### ISSUE TITLE
no-boolean-default should not invalidate default: true

### ISSUE THREAD

**Issue by lamikae** [vuejs/eslint-plugin-vue#820]
**Tell us about your environment**	
* **ESLint version:**	 5.13.0
* **eslint-plugin-vue version:**	 5.2.1
* **Node version:**	10.14.2

**Please show your full configuration:**

```json
  rules: {
    'vue/no-boolean-default': 1,
    ...
  }
```

Some of my component props default boolean values to `true`. It would make sense for the linter to only notify of those that have explicit `default: false` value. With the current rule set they warn about those that should not be warned about.

**Comment by st-sloth** [comment]
But isn't that the whole point of this rule? By default, it forbids the boolean props from having a default value at all. And with option `'default-false'` it only allows the boolean props to have explicit `false` defaults.

**Comment by mysticatea** [comment]
Yes. This is the intentional behavior. The purpose of the rule is to disallow `default: true`.
